### DISCUSSION TITLE
建议/bug：处理mailto链接时可以直接填写收件人

### DISCUSSION THREAD

**Discussion by shenmo7192** [linuxdeepin/developer-center#3121] [Features Request & Ideas | 特性请求 & 头脑风暴]
在使用其他大部分邮件客户端时，在点击到页面上的 mailto链接时，会在打开应用的同时自动填写好收件人，这样可以方便地回信

然而，deepin-mail在处理mailto链接时，仅仅是打开客户端的发信界面，而不会自动填写收件人。

---
这可能是一个bug，在论坛中有人说可以，而我这里不行

这是否与deepin-mail上同时登录了多个账号有关？

https://bbs.deepin.org/post/232260

**Comment by shenmo7192** [comment]
https://github.com/linuxdeepin/developer-center/issues/3123

请关闭此discussion

**Reply by BLumia** [reply]
discussion 就是普通的论坛帖子，不用关闭（（（（

**Reply by shenmo7192** [reply]
> 

这就是ex-bbs.deepin.org吗

**Reply by BLumia** [reply]
国外用户不见得会直接去访问我们 bbs，以及一些需求建议类的 issue 也可以直接转换成 discussion。当然还是有一些好处的，比如需求建议类的可以用标签来区分处理状态什么的
